Merge Conflict/Assets/Scripts/AudioManager/AudioMixerLevels.cs

```diff
@@ -0,0 +1,63 @@
+/**********************************************************************************************************************
+Name:          AudioMixerLevels
+Description:   Handles the volume level of the slider in the audio settings and saves the values in the playerprefs.  
```

Comment:
mention that the script is attached directly to the slider and is not singular manager, such as the AudioManager.

Reply:
done